tools/clang/unittests/HLSLExec/ShaderOpArith.xml

```diff
@@ -4147,11 +4147,25 @@ void MSMain(uint GID : SV_GroupIndex,
         #endif
 
         #ifdef FUNC_WAVE_ACTIVE_ALL_EQUAL
+        inline bool ToggleScalar(bool A) { return !A; }
```

Comment:
I don't think any of these `inline`s are required, and they probably aren't doing what you think they're doing.

Reply:
What do you mean? The goal is to just make sure the value no longer matches. Open to other suggestions.
I was trying to be more creative but then I decided that explicit helpers for each type was an easy way to be confident that the value is changed.

Reply:
I mean the `inline` keyword isn't needed here.